src/session/session_compact.c

```diff
@@ -257,6 +274,11 @@ __compact_worker(WT_SESSION_IMPL *session)
             if (session->op_handle[i]->compact_skip)
                 continue;
 
+            /* Add a 2 second wait to simulate compact slowness. */
```

Comment:
I am curious if we can make it a random number between let's say between 1 - 4 second. Not a big thing though and I am happy with what you have added here.

Reply:
Is 2 seconds sleep enogh?

Reply:
Yeah, I think 2 seconds should be enough because this sleep is under a for loop which runs ~100 times.